### PR TITLE
feat: update flag

### PR DESCRIPTION
**Pull request by rubenVB01**
## Description

In this PR I update the create_bucket flag (default true) to be use_existing_bucket (default false). Also did some documentation changes to align with the rest of the modules.

### CODE REVIEW

File: examples/s3bucket-create-bucket-generate-name/README.md [hunk 1]
```diff
@@ -1,34 +1,48 @@
 # push-based-log-export-s3-bucket - create a new bucket with a default name
 
+_Note: you can see the full source code in the [github repository](https://github.com/terraform-mongodbatlas-modules/terraform-mongodbatlas-push-based-log-export/tree/main/examples/s3bucket-create-bucket-generate-name)_
+
 This example shows how you can use the s3 bucket submodule to enable push-based logging in an Atlas project. The module will create a new s3 bucket with a default name.
 
 
 ## Usage
 
-1. Set the following variable:
```
Comment (EspenAlbert) [inline examples/s3bucket-create-bucket-generate-name/README.md:8]: Is this change intentional?

**Review by EspenAlbert — APPROVED** [review]
LGTM. Would only double-check the indentation of the README.md files

File: examples/s3bucket-create-bucket-generate-name/README.md (hunk 1 above)
Comment (rubenVB01) [reply examples/s3bucket-create-bucket-generate-name/README.md:8]: Yes, don't know why but if I use numbers, in the [terraform public registry](https://registry.terraform.io/modules/terraform-mongodbatlas-modules/push-based-log-export/mongodbatlas/latest/examples/s3bucket-create-bucket-generate-name) it appears strange
